### ISSUE TITLE
Evenly distribute email notifications/ jobs amongst

### ISSUE THREAD

**Issue by MikeBuran** [vinidy/lemon#46]

**Comment by vinidy** [comment]
This is quite a large task which needs to be subdivided. We'll have to make this an epic and break it down.

**Comment by MikeBuran** [comment]
Maybe as an interim step, we could add the ability to claim a post? This would allow one person to be the owner of a task.

**Comment by vinidy** [comment]
Yep, exactly what I was thinking. I need to update the models to support this.
